### DISCUSSION TITLE
Need Help Setting Shortcuts For iOS

### DISCUSSION THREAD

**Discussion by guillaumesoucy94** [Kovah/LinkAce#498] [Support]
Hello, 

I have difficulty setting up a Shortcut for LinkAce in iOS. 

After having following directives from here: https://blog.kovah.de/en/2020/ios-shortcuts-workflow-for-linkace/

My iPhone gave me this error:

![20220629-025740-iOS_Error](https://user-images.githubusercontent.com/22561876/176373230-8a4a950c-ca8b-4792-a98b-c49b3790560d.jpg)
																										
And also, the shortcut did not appear at all in the "Share" menu. 

How to make it works?

My iPhone is using the latest version of iOS.

Thank-you, 

Guillaume

**Comment by guillaumesoucy94** [comment]
Hello, 

Is someone know how to make it work?

Thanks!

Guillaume

**Reply by Kovah** [reply]
I haven't used the shortcut for a long time. Just tried it and it worked well.
Please try to install it again and follow the configuration steps.

**Reply by guillaumesoucy94** [reply]
Hello, 

It looks like if its working now, I can see the LinkAce shortcut but, it gave me an error:
![20220703-124837-LinkAce_iOS](https://user-images.githubusercontent.com/22561876/177049475-05266686-06a7-4f1e-94f3-359b3599595a.jpg)

After verifying, the link get added to LinkAce but why I'm having that error message? 

Also, do you mean that I need reinstall the entire LinkAce instance or only trying to setup the iOS again?

Thank-you, 

Guillaume

**Reply by Kovah** [reply]
Uhm that looks weird. Do you have any sort of firewall in front of LinkAce?

> Also, do you mean that I need reinstall the entire LinkAce instance or only trying to setup the iOS again?

Just the shortcut.

**Reply by guillaumesoucy94** [reply]
Ok will retry.

**Reply by guillaumesoucy94** [reply]
It works now! You was right! Thank-you very much!
![IMG_1963](https://user-images.githubusercontent.com/22561876/178898940-fbe0b892-7ab7-4c35-9c40-c9bd7a543615.jpg)
